.github/workflows/selfdrive_tests.yaml

```diff
@@ -202,6 +202,39 @@ jobs:
       run: |
         eval "$BUILD_CL"
 
+  docker_push_arm64:
+    name: docker push (arm64)
+    runs-on: buildjet-4vcpu-ubuntu-2204-arm
```

Comment:
is there a github actions runner for arm? we are trying to limit our buildjet usage. If not, make sure you put it behind the same flag to prevent forks from being forced to use buildjet

Reply:
maybe qemu https://github.com/docker/setup-qemu-action

Reply:
this would only be run on master, so the usage shouldn't be too high

Reply:
Perhaps we also want to test the arm build on non-master though? So no one breaks it without knowing until they push to master, like if they install an x86 only package